runner/src/server/plugins/engine/pluginHandlers/exit/index.ts

```diff
@@ -63,10 +63,10 @@ export const emailPost = {
     const form = request.pre.form;
 
     const { cacheService, exitService } = request.services([]);
-    const state = await cacheService.getState(request);
```

Comment:
Looks like removing the `getState` call in favour of the `request.pre.state` gives an outdated version of the state without the `exitEmailAddress` value added